src/components/loadingSpinner/loadingSpinner.scss

```diff
@@ -0,0 +1,51 @@
+.loading-spinner-wrapper {
```

Comment:
Taka mała uwaga, jak już korzystamy z SASSa, to już go używajmy. Wykorzystaj tutaj operatorr &, żeby nie pisać cały czas loading spinner.